src/test/Navbar.test.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { render, fireEvent } from '@testing-library/react';
+import { Provider } from 'react-redux';
+import configureStore from 'redux-mock-store';
```

Comment:
- [ ] This is why your `Nav.test.js` is not passing. You have required/imported a package or project dependency that is not in your code therefore making it impossible to run perfectly. To fix this, I recommend that you run `npm i redux-mock-store` on your terminal or if you have, kindly push it to GitHub.

![image](https://github.com/MargaretTechworld/React-Redux-Solo-Capstone/assets/99715640/04de9c8d-4b53-4e20-a65d-f9a54d2a6b62)

- Dependency missing in your `package.json` :point_down: 
![image](https://github.com/MargaretTechworld/React-Redux-Solo-Capstone/assets/99715640/0c0cffd9-5ba3-4043-8b65-a5ab6fb55e59)